### ISSUE TITLE
Polished pages

### ISSUE THREAD

**Issue by Clamvi** [Seytek/project_czu#6]
It remains to polish the pages. Otherwise, I think it's already good.
We have 6 html pages. There are two more left if I've not mistaken. I will take care of the design of the event page.

Franta, I will ask you to make a page with the departments of the university (list of departments: chemistry, biology, architecture, etc.) 
link to departments.html will be in education.html.

**Comment by Clamvi** [comment]
just a simple page with a list of departments, you can add something more its up to you

**Comment by Clamvi** [comment]
I was thinking of combining css files into one, because I noticed that everyone gets their own version of the basics of the site (header, footer, container).
I'll make it if you think it's worth doing

**Comment by sch3fr** [comment]
> just a simple page with a list of departments, you can add something more its up to you

will do 👍

> I was thinking of combining css files into one, because I noticed that everyone gets their own version of the basics of the site (header, footer, container). I'll make it if you think it's worth doing

Yeah I was also thinking we have a lot of redundant code, IMO it's worth it to simplify it

**Comment by Clamvi** [comment]
So I got rid of extra css files.
Now you can write your styles in style.css(aka main.css was renamed). But be careful and try not to repeat the names of past classes, so as not to break everything.

**Comment by Clamvi** [comment]
7eedc151fc4e6eff860a48f402672879c40de992
if it looks empty, try to recreate or take something from other university websites. theres no need to make everything by yourself

**Comment by Clamvi** [comment]
- [ ]  Guys, please fix all div tags with classes .section__title, in general, wherever there is a prefix "title".  If there is div make it h1-h6. The same applies to the other unsuitable tags

- [x] and please edit the page event.html I don't have time at all right now ;(, I'm sorry. I'll upload what I have so far.

**Comment by sch3fr** [comment]
I've just looked through all the html files. Headers are fixed now. I'll look through my files for incorrect tags now. I'll also try doing something with event.html, but that may be finished tomorrow morning.

EDIT: Checked my pages, no more incorrect tags in campus.html, departments.html and news.html
EDIT2: Tried doing something with the event.html, not sure if you'll like it tho, seems kinda weird so far
